### PR TITLE
[PRT] Backport improvements to the records module from 8.0

### PR DESCRIPTION
**Pull request by StefanRijnhart**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+*.pyc
```
Comment (pedrobaeza) [inline .gitignore:1]: Don't we want a more ellaborated gitignore like the one in other repos?

File: bin/addons/openupgrade_records/model/comparison_config.py [hunk 2]
```diff
@@ -23,11 +23,25 @@
...
+    from openerp.addons.openupgrade_records.lib import apriori
+except ImportError:
+    from openupgrade_records.lib import apriori
+
+try:
+    from openerp.osv.orm import Model, except_orm
+    from openerp.osv import fields
+    from openerp.tools.translate import _
+except ImportError:
+    from osv.osv import osv as Model, except_osv as except_orm
```
Comment (pedrobaeza) [inline bin/addons/openupgrade_records/model/comparison_config.py:39]: We know that this is going to fall always in this part for v5, but good strategy to keep all versions with the same code :wink:

**Comment by pedrobaeza** [comment]
The rest LGTM

File: .gitignore (hunk 1 above)
Comment (StefanRijnhart) [inline .gitignore:1]: I'm not that interested. As it contains 'lib', which is a directory in this module, it has so far caused more problems than it has solved.

File: bin/addons/openupgrade_records/model/comparison_config.py (hunk 2 above)
Comment (StefanRijnhart) [inline bin/addons/openupgrade_records/model/comparison_config.py:39]: Exactly. That allowed me to simply copy the code over from 8.0 and it worked (if I remember correctly).

**Comment by pedrobaeza** [comment]
:+1:

**Comment by pedrobaeza** [comment]
Should we proceed with the merge with this old one?

**Comment by StefanRijnhart** [comment]
Yes, that should be good.

**Pull request merged**